test/basic.html

```diff
@@ -43,10 +37,11 @@
     </template>
   </test-fixture>
 
-  <script type="module">
+<script type="module">
```

Comment:
Need to `rm -rf node_modules package-lock.json && npm i && npm run format` to pick up https://github.com/PolymerLabs/webmat/pull/9#event-1705191584 (just released), otherwise module scripts aren't getting auto-formatted.

Reply:
Updated package-lock.json and formatted.